src/pages/docs/guides/chat/build-livestream.mdx

```diff
@@ -0,0 +1,210 @@
+
+<Code>
+```javascript
+if (timeSinceLastMessage > minTimeBetweenMessages) {
+  room.messages.send({text: 'my message'});
+} else {
+  throw new Exception('Too many messages');
+}
+```
+</Code>
+
+### Message rate consistency
+
+Whether the rate of messages being sent is consistent or not for your use case can be a determining factor.
+
+If the message rate only peaks to high levels during exciting moments on stream, such as a point being scored in a sporting event, then this can be acceptable for some use cases. In these instances then features such as room-wide reactions help to provide the general sentiment of the chat when users can't read every message.
+
+## Core functionality
+
+Core functionality includes the features that are required to build a livestream chat application.
+
+### Authenticate users
+
+The simplest way to handle authentication in a livestream chat application is using [JSON Web Tokens (JWTs)](/docs/auth/token#jwt).
+
+Users need to assume an identity when using Ably Chat. If your application requires a login then this can be a name, or identifier, read from your identity provider. If your application allows anonymous users, then use a mechanism to randomly assign users a unique client ID when issuing a JWT.
+
+#### Login required
+
+If a user is required to log in, use their unique user identifier as the client ID when they authenticate with Ably. This will enable you to easily associate their usage in your application with their activity over Ably.
+
+#### No login required
+
+If your application does not require users to log in, assign a randomly generated client ID to users when they authenticate with Ably.
+
+### Send messages
+
+There's nothing different to consider when [sending messages](/docs/chat/rooms/messages#send) in a livestream chat compared to how you would normally send a message. However, Ably recommends enabling server-side batching for livestream chat applications.
+
+[Server-side batching](/docs/messages/batch#server-side) means that Ably will group published messages over an interval before publishing them to subscribers.This will help to reduce costs, as streams usually contain a large number of talkative users.
+
+Be aware of the length of the interval you configure, over which messages are batched. If it is too high, then users may notice the delay from messages being sent, however this is normally less of a concern for fast-paced livestreams. An interval of 1-2 seconds is likely sufficient for most livestream chat applications.
+
+#### Message ordering
+
+Messages will be received by users in the order that they are broadcast, by the region that the user is connected to. This may differ slightly to users in other regions, which for a fast-paced livestream is usually acceptable.
+
+If you want to ensure that every user sees every message in exactly the same order, you can use the [global ordering of messages](/docs/chat/rooms/messages#global-ordering) to do this.
```

Comment:
Perhaps say here "... that every user sees messages displayed in exactly the same order"

The code below doesn't affect realtime order, it just changes how you'd order messages in your applications view layer